clearwater-cassandra/usr/share/clearwater/bin/poll_cassandra.sh

```diff
@@ -35,12 +35,18 @@
 # as those licenses appear in the file LICENSE-OPENSSL.
 
 # Get how long the cassandra process has been running. If this is less than
-# 2 minutes, then don't poll cassandra (as it may not be up yet).
+# 2 minutes, then don't poll cassandra (as it may not be up yet), unless
+# we're specifically asked to.
 # Getting the uptime can fail if the cassandra process fails - this is caught
 # by the monit script.
-value=$( ps -p $( cat /var/run/cassandra/cassandra.pid ) -o etimes=)
-if [ $? == 0 ] && [ "$value" -lt 1200 ]; then
-  exit 0
+
+[ $# -le 1 ] || { echo "Usage: poll_cassandra [--no_grace_period] (defaults to a two minute grace period)" >&2 ; exit 2 ; }
```

Comment:
Although here it looks like you could literally type anything as your argument so perhaps a bit of a moot point